gcp/observability/logging_test.go

```diff
@@ -131,8 +133,13 @@ func (s) TestClientRPCEventsLogAll(t *testing.T) {
 		},
 		FullDuplexCallF: func(stream grpc_testing.TestService_FullDuplexCallServer) error {
 			for {
```

Comment:
I don't think this actually wants to be in a loop.  The client just sends once, right?

Reply:
Ah, lol fair point :). Lol unnecessary while true.